erlang/eqwalizer/Portfile

```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               github 1.0
+PortGroup               java 1.0
+
+github.setup            WhatsApp eqwalizer 0.25.3 v
+revision                0
```

Comment:
you should set `github.tarball_from` to the appropriate values here as well. Ideally `releases` is upstream provides them, otherwise `archive`